Added troubleshooting documentation (#194)
vmware-labs/service-bindings: README.md

kapp delete -a service-bindings
```

## Troubleshooting

> Service Bindings for Kubernetes is an implementation of the Service Binding Specification for Kubernetes, If any trouble shows up please refer to the troubleshooting guide [here](./docs/troubleshooting.md).

Reviewer: ```suggestion
For basic troubleshooting Service Bindings, please see the troubleshooting guide [here](./docs/troubleshooting.md).
```

Author: Changed
